src/api/package/metadata.ts

```diff
@@ -8,6 +8,9 @@ export interface Metadata {
   };
 }
 
+/**
+ * Fetch metadata of a specific package from the backend.
+ */
```

Comment:
I believe there's an eslint rule that enforces jsdoc / docstrings. May be worth adding in a future PR so we can enforce this?

Reply:
Yeah, i'll take a note but leave it for now.